app/test/publisher/backend_test.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:gcloud/db.dart';
+import 'package:test/test.dart';
+
+import 'package:pub_dartlang_org/account/backend.dart';
+import 'package:pub_dartlang_org/publisher/backend.dart';
+import 'package:pub_dartlang_org/publisher/models.dart';
+
+import '../frontend/utils.dart';
+import '../shared/test_services.dart';
+
+void main() {
+  group('PublisherBackend', () {
```

Comment:
Do we really want to test this? Can't we cover all this by hitting the API end-points? That way we won't have to refactor tests when/if we refactor the implementation.

I mean the REST API is something we can aim to keep stable, and if we get the same test coverage by hitting it, then why do we need low level tests. Can we get the same test coverage is a fair question though? :)

Reply:
Fair point, but at the same time we could just migrate these tests to use API endpoints instead of directly calling the methods here.

Reply:
Sure, tht works too...